reference/forms/types/collection.rst

```diff
@@ -363,6 +364,29 @@ rendering your form, having the entire "form row" may be easier for you.
 For details on how to actually use this option, see the above example as
 well as :ref:`cookbook-form-collections-new-prototype`.
 
+prototype_data
+~~~~~~~~~~~~~~
+
+**type**: ``mixed`` **default**: ``null``
+
+Allows you to define a specific data for the prototype. Each new row added will
+initially contain the data set by this option.
+By default, the same data as defined with ``entry_options`` for other entries
+will be used.
+
+.. code-block:: php
+
+    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+    $builder->add('tags', CollectionType::class, array(
+        'class' => \AppBundle\Entity\Tag::class,
```

Comment:
For the Symfony 2.8 docs we always use a string to be compatible with PHP 5.3 and 5.4.